make-sync-function

```diff
@@ -20,7 +20,7 @@ if (process.argv.length !== 4) {
 
   console.log('Arguments:');
   console.log('    sync_document_definitions_file');
-  console.log('        Path to the JavaScript/JSON file that defines the document types to use in the sync function');
+  console.log('        Path to the JavaScript file that defines the document types to use in the sync function');
```

Comment:
This description was misleading. Currently it's impossible to define a document definitions file as pure JSON since each document type must define a type filter function, which is not supported by JSON.